ltsa/get_keycloak_token.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/ksh
+
+# Reusable script to get Keycloak access token
+# Usage: ACCESS_TOKEN=$(./get_keycloak_token.sh)
+# Or: source ./get_keycloak_token.sh
+
+# Configuration - Read from environment variables or use defaults
+KEYCLOAK_URL="${KEYCLOAK_URL:-https://epd-keycloak-dev.apps.silver.devops.gov.bc.ca/auth}"
+REALM="${KEYCLOAK_REALM:-forms-flow-ai}"
+CLIENT_ID="${KEYCLOAK_CLIENT_ID:-site-service}"
+CLIENT_SECRET="${KEYCLOAK_CLIENT_SECRET}"
+
+# Function to log to stderr
+log() {
+    print "$1" >&2
+}
+
+# Check if required environment variables are set
+if [[ -z "$CLIENT_SECRET" ]]; then
```

Comment:
I would recommend, we remove defaults and validate all the inputs, since defaults can create problems in the ministry servers when actual values are not provided.